articles/Remote Debugging/how-to-remote-debugging-with-vorlonJS.md

```diff
@@ -0,0 +1,33 @@
+<!--
+   tags:
+       Debugging:
+           - "Remote debugging with VorlonJS"
+-->
+# Remote debugging with VorlonJS
+
+> This article explains how to remotely debugging and testing your JavaScript using `vorlon.js`. While working with a Mac and an iPad might be straight forward, achievieving the same thing with a PC is more complicate. Here is one possible approach.
+
+## Server (your local machine)
+
+Install VorlonJS server from npm
+```console
+$ npm i -g vorlon
```

Comment:
I think we normally use `bash` to identify the language of such a code block. `console` might work, it might not.

I think we prefer not to include the `$` before the code line. Dropping it also makes it easier to copy & paste the line.